channels/generic/websockets.py

```diff
@@ -162,11 +162,58 @@ def send(self, content, close=False):
         """
         Encode the given content as JSON and send it to the client.
         """
-        super(JsonWebsocketConsumer, self).send(text=json.dumps(content), close=close)
+        super(JsonWebsocketConsumer, self).send(text=self.encode_json(content), close=close)
+
+    @classmethod
+    def decode_json(cls, text):
+        return json.loads(text)
+
+    @classmethod
+    def encode_json(cls, content):
+        return json.dumps(content)
 
     @classmethod
     def group_send(cls, name, content, close=False):
-        WebsocketConsumer.group_send(name, json.dumps(content), close=close)
+        WebsocketConsumer.group_send(name, cls.encode_json(content), close=close)
+
+
+class WebsocketMultiplexer(object):
```

Comment:
FYI, this is almost entirely just a move of this class (since it's referenced inside `WebsocketDemultiplexer`)...